test/integration_test.go

```diff
@@ -150,6 +150,10 @@ func (ts *IntegrationTestSuite) SetupTest() {
 		options.EnableSessionWorker = true
 	}
 
+	if strings.Contains(ts.T().Name(), "LocalActivityWorkerOnly") {
+		options.LocalActivityWorkerOnly = true
+	}
+
```

Comment:
nit: This is already being done above, but it'd be nice if we could somehow pass this in from the test itself rather than have setup inspect the test name.

Reply:
I agree, it looks hacky, but I don't see a cleaner way of doing this. Will probably leave it as is for now.